opm/grid/CpGrid.hpp

```diff
@@ -1225,7 +1225,7 @@ namespace Dune
         ///         perforated cells local to the process, for all wells (sorted by name)
         std::pair<bool,std::vector<std::pair<std::string,bool>>>
         loadBalance(const std::vector<cpgrid::OpmWellType> * wells,
-                    const std::unordered_map<std::string, std::set<std::array<int,3>>>* possibleFutureConnections = nullptr,
+                    const std::unordered_map<std::string, std::set<int>>* possibleFutureConnections = nullptr,
```

Comment:
Same question here as in opm-simulators. Would this also work:
```suggestion
                    const std::unordered_map<std::string, std::set<int>>& possibleFutureConnections = {},
```
and internally you can test the size?